File: 8KYU/Geometry Basics Distance between points in 2D/solution.cs
using System;
public class Kata
{
  public static double DistanceBetweenPoints(Point a, Point b)
  {
    var deltaX = Math.Abs(a.X - b.X);
    var deltaY = Math.Abs(a.Y - b.Y);
    return Math.Sqrt((deltaX * deltaX) + (deltaY * deltaY));
  }
}

File: 8KYU/Geometry Basics Distance between points in 2D/solution.c
#include <stdlib.h>
#include <math.h>

typedef struct Point {
    double x;
    double y;
} point;

double distance_between_points(point a, point b) {

    float dx = fabs(a.x - b.x);
    float dy = fabs(a.y - b.y);
    return sqrt((dx * dx ) + (dy * dy));
}